.gitignore

```diff
@@ -0,0 +1,24 @@
+/.cache
+/.work
+/_output
+/config/
+/config
+cover.out
+/vendor
+/.vendor-new
+.DS_Store
+
+
+# gitlab example
+# exclude files generate by running the example
+external-dns-*.tgz
+gitlab-*.tgz
+gitlab-gcp.yaml
+gitlab/
+
+# ignore vscode files (debug config etc...)
+/.vscode
```

Comment:
hmm, it's odd that this `.gitignore` file has an entry for `.vscode` but we're still seeing that folder as included in this PR, maybe because it was already part of the index before you added it to the `.gitignore`? 🤔 

Do you want to try to remove the `.vscode` folder again? I think this answer on stackoverflow will help accomplish that: https://stackoverflow.com/a/9750651  🙏

Reply:
Resolved.  Please review and merge at your convenience.